rust/perspective-viewer/src/less/status-bar.less

```diff
@@ -280,20 +276,45 @@
                     background-color: var(--plugin--background);
                 }
 
+                // The label.
+                @mixin label {
+                    display: block;
+                    position: absolute;
+                    top: calc(100% + 3px);
+                    left: 50%;
+                    translate: -50%;
+                    margin: 0;
+                    padding: 5px;
+                    height: auto;
+                    white-space: nowrap;
+                    line-height: 1;
+                    background-color: var(--icon--color);
+                    border-radius: 3px;
+                }
                 & > span {
-                    display: contents;
+                    @include label;
                 }
 
                 & > .dropdown-width-container {
+                    @include label;
                     overflow: visible;
-                    display: inline;
                     pointer-events: all;
                     & > select {
+                        position: static;
                         cursor: pointer;
-                        margin-left: -28px;
-                        padding-left: 28px;
-                        width: calc(100% + 28px);
-                        height: 36px;
+                    }
+
```

Comment:
Pseudo-element positioned in gap allows cursor to traverse from toolbar button to dropdown without losing hover.